addons/account/views/account_move_views.xml

```diff
@@ -1168,7 +1168,11 @@
                                         <field name="discount" width="50px" string="Disc.%" optional="hide"/>
                                         <field name="tax_ids" widget="many2many_tax_tags"
                                                domain="[('type_tax_use', '=?', parent.invoice_filter_type_domain), ('company_id', 'parent_of', parent.company_id), ('country_id', '=', parent.tax_country_id)]"
-                                               context="{'append_type_to_tax_name': not parent.invoice_filter_type_domain, 'active_test': True, 'dynamic_fiscal_position_id': parent.fiscal_position_id}"
+                                               context="{
+                                                    'append_fields': ['type_tax_use'],
```

Comment:
We don't want this for sales and purchases journals because it is already obvious which type we have

Reply:
100%